components/pickerbutton/stories/pickerbutton.stories.js

```diff
@@ -108,35 +114,178 @@ export default {
 		isQuiet: false,
 		isDisabled: false,
 		isFocused: false,
-		isKeyboardFocused: false,
 		iconType: "ui",
 		iconName: "ChevronDown",
-		position: "right"
+		position: "right",
 	},
 };
 
+// Story specific templates
+const SizingTemplate = (args) => html`
+	<div
```

Comment:
I think the preference is to get markup like this into the `template.js` file as an export (Cassondra left me some requests regarding that on #2793 & #2833 for things like the variant checkbox group template and the divider sizes templates.)

Reply:
I've moved these into the `template.js`.